### PR TITLE
add fetch logs to nightly build fetch integrations

### PR DESCRIPTION
**Pull request by jbabazadeh**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Related:https://jira-hq.paloaltonetworks.local/browse/CIAC-5870

## Description
Added fetch logs to nightly build integrations that have fetch

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py [hunk 1]
```diff
@@ -1454,6 +1454,9 @@ def fetch_incidents(client: Client, max_incidents: int,
 
         if last_fetch:
             if incident_created_time < last_fetch:
+                skip_incidents += 1
```
Comment (yuvalbenshalom) [inline Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py:1457]: you cont them, but do not print.
print the skipped incidents

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
looks good.
see my comment.

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
the same combat about id and time like in the other PR.

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py [hunk 2]
```diff
@@ -2171,10 +2172,15 @@ def fetch_incidents(client: Client) -> list:
 
         try:
             if datetime.strptime(ticket[client.timestamp_field], DATE_FORMAT) < snow_time_as_date:
+                skipped_incidents += 1
+                demisto.debug(
+                    f"""ServiceNowV2 - -Skipping incident with sys_id={ticket.get('sys_id')} because its creation time is
+                    smaller than the last fetch.""")
```
[the remark this answers is not in the document]
Comment (jbabazadeh) [reply Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py]: DONE

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**